### ISSUE TITLE
'koch docs' fails on Windows

### ISSUE THREAD

**Issue by rockcavera** [nim-lang/Nim#13860]
'koch docs' fails on Windows.

### Example
```
koch docs
```

### Current Output
https://pastebin.com/eC31PTwT

### Expected Output
Let it end without failure.

### Additional Information

* Works correctly on Linux. Same version, that is, devel.

```
nim --version
Nim Compiler Version 1.1.1 [Windows: amd64]
Compiled at 2020-04-03
Copyright (c) 2006-2020 by Andreas Rumpf

git hash: 920add5880e74695025d2383dd6806c51bd831b5
active boot switches: -d:release
```

**Comment by Araq** [comment]
Works for me. Disable your AntiVir.

**Comment by rockcavera** [comment]
> Works for me. Disable your AntiVir.

I don't use any antivirus and windows defender is disabled.

This installation is by devel, updated by 'git pull' and compiled with 'build_all.bat'.

**Comment by rockcavera** [comment]
I managed to make 'koch docs' work on Windows, but only when I installed Nim on 'C:\Nim'. I did the same procedures on 'D:\Nim' and was unsuccessful, even with a clean installation. After noticing this, I decided to do the installation on a third E partition and it worked correctly.

Then I noticed that in 'D:\' there was an old nim backup file 'nim.cfg' that changed the nimcache. After I deleted it, it worked correctly.

I could say that the problem is solved.
